site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -48,9 +48,9 @@ export const Language = {
   filterName: "Filters",
   yourWorkspacesButton: "Your workspaces",
   allWorkspacesButton: "All workspaces",
-  workspaceTooltipTitle: "What is workspace?",
+  workspaceTooltipTitle: "What is a workspace?",
   workspaceTooltipText:
-    "It is your workstation. It is a workspace that will provide you the necessary compute and access to your development environment.",
+    "A workspace is your development environment in the cloud. It includes the compute infrastructure and tools you need to work on your project",
```

Comment:
```suggestion
    "A workspace is your development environment in the cloud. It includes the infrastructure and tools you need to work on your project.",
```

A workspace includes more than compute infrastructure and a period was missing.